### PR TITLE
[ET-1899] Feat- Remove header image from series pass settings.

### PR DESCRIPTION
**Pull request by rafsuntaskin**
### 🎫 Ticket

[ET-1899] 

### ✔️ Checklist
- [x] I've included a changelog entry in the readme.txt file. 
- [x] My code is tested. 
- [x] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

[ET-1899]: https://stellarwp.atlassian.net/browse/ET-1899?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Tickets/Flexible_Tickets/Series_Passes.php [hunk 1]
```diff
@@ -281,8 +283,8 @@ public function unregister(): void {
 			$this,
 			'filter_tickets_attendees_report_js_config'
 		] );
-
 		remove_filter( 'tribe_template_after_include:tickets/v2/tickets/title', [ $this, 'render_series_passes_header_in_frontend_ticket_form' ], 10, 3 );
+		remove_filter( 'tribe_template_pre_html:tickets/admin-views/editor/panel/header-image', [ $this, 'hide_header_image_option_from_ticket_settings' ], 10, 5 );
```
Comment (lucatume) [inline src/Tickets/Flexible_Tickets/Series_Passes.php:287]: The way you're removing the header image on Series is correct, but the `Series_Passes` controller is not the right place to do it.

The task is to remove the Ticket header image from the Series **post**, not the Series_Passes **ticket**.
For this reason, I think this change  should be done in the `Base` controller, not this one.

Comment (rafsuntaskin) [reply src/Tickets/Flexible_Tickets/Series_Passes.php]: Agreed, moved it now. I started the code from the series pass perspective and hence used that controller.

File: src/Tickets/Flexible_Tickets/Series_Passes.php [hunk 2]
```diff
@@ -281,7 +281,6 @@ public function unregister(): void {
 			$this,
 			'filter_tickets_attendees_report_js_config'
 		] );
-
```
Comment (lucatume) [inline src/Tickets/Flexible_Tickets/Series_Passes.php:284]: Revert changes to this file completely.

File: src/Tickets/Flexible_Tickets/Base.php [hunk 3]
```diff
@@ -545,4 +549,25 @@ public function filter_series_type_label( $type ): string {
...
 	}
+
+	/**
+	 * Filters the HTML for the ticket editor to hide the header image option from the ticket settings.
+	 *
+	 * @since TBD
+	 *
+	 * @param null|string           $html       The initial HTML.
+	 * @param string                $file       Complete path to include the PHP File.
+	 * @param array                 $name       Template name.
```
Comment (lucatume) [inline src/Tickets/Flexible_Tickets/Base.php:560]:
```suggestion
	 * @param string[]                $name       Template name.
```

**Review by lucatume — CHANGES_REQUESTED** [review]

**Pull request merged**